frontend/components/AddSite/TemplateSiteList/index.js

```diff
@@ -62,30 +65,30 @@ class TemplateList extends React.Component {
 
     let index = 0;
 
-    const templateGrid = templateRows.map((row, rowIndex) => {
-      return (
-        <div className="usa-grid" key={rowIndex}>
-          {row.map((templateName, cellIndex) => {
-            const template = templates[templateName];
-            return (
-              <div
-                  className={`usa-width-one-${cellSize}`}
-                  key={cellIndex}>
-                <TemplateSite
-                  name={templateName}
-                  index={index++}
-                  thumb={template.thumb}
-                  active={this.state.activeChildId}
-                  handleChooseActive={this.handleChooseActive}
-                  handleSubmit={this.props.handleSubmitTemplate}
-                  defaultOwner={this.props.defaultOwner}
-                  {...template} />
-              </div>
-            );
-          })}
-        </div>
-      );
-    });
+    const templateGrid = templateRows.map(row => (
```

Comment:
mostly linting fixes here. I changed the `key` properties so that they wouldn't be loop-index-based as well.